src/Microsoft.VisualStudio.Editor.Razor/RazorDocumentManager.cs

```diff
@@ -2,12 +2,13 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Host;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
 
 namespace Microsoft.VisualStudio.Editor.Razor
 {
-    internal abstract class RazorDocumentManager
+    internal abstract class RazorDocumentManager : ILanguageService
```

Comment:
Shower Thought: A general rule of thumb with the approach we're taking in Razor is, if we have an `internal abstract class` then it should always be an `ILanguageService` 😄